src/server/adminJs/tabs/donationTab.ts

```diff
@@ -265,6 +269,55 @@ export const buildDonationsQuery = (
   return query;
 };
 
+export const FillPricesForDonationsWithoutPrice = async () => {
```

Comment:
@RamRamez  Can you add a test case for this function?
Create a donation and ,ake the valueUsd of donation as null
Then check if the price has filled or not

Reply:
You're right. I added a test case.
Thanks @mohammadranjbarz